test-jw/src/app.css

```diff
@@ -7,4 +7,8 @@
 
 body {
     height: 200vh;
-}
\ No newline at end of file
+}
+
+.hidden {
```

Comment:
Consider changing the class name to `forcefullyHidden` or `fandomHidden`, so it's less unlikely it'll get overwritten by our users/wikis custom CSS. Recently, there was an issue in Ad Engineering that the custom CSS was overwritting AdEngin's `.hidden` class - let's learn from it and as long as we allow custom CSS let's avoid common CSS class names.

Reply:
Agreed 👍